packages/vuetify/src/composables/elevation.ts

```diff
@@ -4,7 +4,6 @@ import propsFactory from '@/util/propsFactory'
 // Types
 export interface ElevationProps {
   elevation?: number | string | null
-  flat?: boolean
```

Comment:
not 100% sold on removing flat, but `elevation="0"` is more declarative. taking feedback

Reply:
I'm for keeping flat as an alias to elevation="0".

Reply:
For now the elevation composable will dictate the interface available. We will revisit this after aliasing is hooked up.